### PR TITLE
[sil_tai_dam-help] Add help file

### PR DESCRIPTION
**Pull request by LornaSIL**

### CODE REVIEW

**Comment by darcywong00** [comment]
The OSK originally didn't render for me until I manually installed `release/shared/fonts/sil/tai_heritage_pro/TaiHeritagePro-Bold.ttf`.

Is it preferable to 
1. Use screenshots or
2. Deliver the web fonts in the page?

**Comment by LornaSIL** [comment]
Oh, yes. Maybe I should do a screenshot. I don't think we're using webfonts
yet, are we?

**Comment by DavidLRowe** [comment]
Or export OSK from Keyman Developer to .png?

File: release/sil/sil_tai_dam/source/help/sil_tai_dam.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  <td width=67 valign=center style='width:117.0pt;border:none;border-bottom:solid windowtext 1.0pt;
+  padding:0pt 0pt 0in 0pt'>
+  <p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span class=CharacterCode2000><span 
+  style='font-size:12.0pt;font-family:"Tai Heritage Pro;font-weight:normal'>&#x003C;</span></span></p>
+  </td>
+  <td width=72 valign=center style='width:.75in;border-top:none;border-left:none;
+  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 2.25pt;
+  padding:0pt 0pt 0in 0pt'>
+  <p class=MsoNormal align=left style='text-align:center;page-break-after:avoid'><span
+   style='font-size:12.0pt;font-family:"Tai Heritage Pro"'>|&lt</span></p>
```
Comment (darcywong00) [inline release/sil/sil_tai_dam/source/help/sil_tai_dam.php]: minor thing: lines 100 and 119 need semicolons
```
|&lt;
|&gt;
```

**Comment by LornaSIL** [comment]
https://help.keyman.com/keyboard/sil_tai_dam

**Pull request merged**